tests/StaticCaching/NoCacheSessionTest.php

```diff
@@ -171,6 +171,17 @@ public function it_ignores_the_query_string()
         $this->assertEquals('http://localhost/test', $session->url());
     }
 
+    #[Test]
+    public function it_normalizes_query_string()
+    {
+        $this->get('/test?utm_source=linkedin.com&utm_medium=referral&utm_campaign=foo');
```

Comment:
When I comment out my changes in the `ServiceProvider` this test continues to pass.

It looks like Laravel is normalizing the query string itself in tests, which doesn't happen when I'm testing using Herd.